### PR TITLE
Update debug-cluster and crictl files to get them consistent with the…

### PR DESCRIPTION
**Pull request by DanyC97**
… style guidelines

Two tasks from https://github.com/kubernetes/website/issues/12740 were covered:

- [x] https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/
- [x] https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/

question for sig-docs members:

should we rename the task title `title: "Monitor, Log and Debug"` to `Monitoring, Logging and Debugging` to keep alignment/ consistency ?

### CODE REVIEW

**Comment by DanyC97** [comment]
@cody-clark please help review, thank you

File: content/en/docs/tasks/_index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Run Jobs using parallel processing.
 
 Configure load balancing, port forwarding, or setup firewall or DNS configurations to access applications in a cluster.
 
-## Monitoring, Logging, and Debugging
+## Monitoring, Logging and Debugging
```
Comment (cody-clark) [inline content/en/docs/tasks/_index.md]: I'm reluctant to approve getting rid of a serial comma... XD

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 2]
```diff
@@ -36,28 +36,28 @@ of the relevant log files.  (note that on systemd-based systems, you may need to
...
-   * /var/log/kube-proxy.log - Kube Proxy, responsible for service load balancing
+   * `/var/log/kubelet.log` - Kubelet, responsible for running containers on the node
+   * `/var/log/kube-proxy.log` - Kube Proxy, responsible for service load balancing
 
 ## A general overview of cluster failure modes
 
 This is an incomplete list of things that could go wrong, and how to adjust your cluster setup to mitigate the problems.
 
-Root causes:
+##### Root causes:
```
Comment (cody-clark) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: An H5 seems very specific. Was there a reason to go this deep?

Comment (DanyC97) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: rational was to get to articulate the 3 pillars: `Root causes`, `Specific scenarios` & `Mitigations` from the rest of the actions/ text.

I tried to just make the text bold but in my local preview website it didn't make the difference i was expecting hence h5.

Any alternative ?

File: content/en/docs/tasks/_index.md (hunk 1 above)
Comment (DanyC97) [reply content/en/docs/tasks/_index.md]: the reason here was purely grammer, nothing else. and i tried to align this file with `content/en/docs/tasks/debug-application-cluster/_index.md`.

At the same time i've also asked in the PR description if 

> should we rename the task title title: "Monitor, Log and Debug" to Monitoring, Logging and Debugging to keep alignment/ consistency ?

Comment (cody-clark) [reply content/en/docs/tasks/_index.md]: Renaming the task title for consistency is a good idea.

I'm still not sold on deleting the the serial comma. "Monitoring, Logging, and Debugging" is just as correct as "Monitoring, Logging and Debugging". And I'm not ready to instigate the Oxford comma debate in sig-docs!

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md (hunk 2 above)
Comment (cody-clark) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: I completely agree that those pillars should be differentiated. Don't H3s work here? 

I prefer semantic markup for hierarchy over appearance. There's also the accessibility consideration: https://www.w3.org/WAI/tutorials/page-structure/headings/

> Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a <h2> is not followed directly by an <h4>, for example.

Comment (DanyC97) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: ack, will give it a go and then you will see if it makes sense with h3 as part of the review

Comment (DanyC97) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: @cody-clark please help review again, made the change from h5 to h3

**Comment by DanyC97** [comment]
no problem at all @cody-clark , i'm all good

**Pull request merged**